traffic_ops/traffic_ops_golang/wrappers.go

```diff
@@ -117,7 +117,12 @@ func (a AuthBase) GetWrapper(privLevelRequired int) Middleware {
 				handleErr(http.StatusInternalServerError, errors.New("No config found"))
 			}
 
-			currentUserInfo := auth.GetCurrentUserFromDB(DB, a.getCurrentUserInfoStmt, username, time.Duration(cfg.DBQueryTimeoutSeconds)*time.Second)
+			currentUserInfo, userErr, sysErr, code := auth.GetCurrentUserFromDB(DB, a.getCurrentUserInfoStmt, username, time.Duration(cfg.DBQueryTimeoutSeconds)*time.Second)
+			if sysErr != nil {
```

Comment:
Should this be `if userErr != nil || sysErr != nil {` ?

Reply:
I debated on that and went with having the function always return a sysError to log, i.e. in the case of no rows being returned it logs that, but I think checking that way feels safer since api.HandleErr handles a nil sysErr just fine.